integration-tests/runtime-tests/test/tests/dexTests/constantProductDex/testHandlers/constantProductDex.ts

```diff
@@ -0,0 +1,166 @@
+import { sendAndWaitForSuccess } from '@composable/utils/polkadotjs';
+import {KeyringPair} from "@polkadot/keyring/types";
+import { u128 } from '@polkadot/types-codec';
+
+/**
+ *Contains handler methods for the constantProductDex Tests. 
+ */
+let poolId: number;  
+let constantProductk: number;
+let baseAmountTotal: number;
+let quoteAmountTotal: number;
+let mintedLPTokens: number;
+baseAmountTotal = 0;
+quoteAmountTotal = 0;
+mintedLPTokens = 0;
+
+
+//In order to successfully send funds, the users should have sufficient funds with assetId's 2 and 3 respetively.
+export async function mintAssets(walletId:KeyringPair, assetId: number, amount: number){
```

Comment:
This function can be removed now right?

Reply:
i do not fund it in rust codebase - mintInto

Reply:
This was a util function to mint assets into the accounts, but @Dom-Roth had already created one, I used that one. Removed.